ddp-workspace/projects/ddp-sdk/src/lib/components/activityForm/answers/activity-composite-answer/activityCompositeAnswer.component.html

```diff
@@ -14,6 +14,7 @@
                 [block]="childBlock"
                 [readonly]="readonly"
                 [validationRequested]="validationRequested"
+                [studyGuid]="config.studyGuid"
```

Comment:
The config.studyGuid is, for all practical purposes, a read-only global. If you need it, just get by injecting the config service were needed.

Reply:
ok